src/rSW_Site.c

```diff
@@ -639,8 +639,8 @@ void onSet_SW_SIT(SEXP SW_SIT) {
 
 	/* check for any discontinuities (reversals) in the transpiration regions */
 	for (r = 1; r < v->n_transp_rgn; r++) {
-		if (_TranspRgnBounds[r - 1] >= _TranspRgnBounds[r]) {
-			LogError(logfp, LOGFATAL, "siteparam.in : Discontinuity/reversal in transpiration regions.\n");
+		if (v->_TranspRgnBounds[r - 1] >= SoilWatAll.Site._TranspRgnBounds[r]) {

```

Comment:
Why use once `v` and once `SoilWatAll.Site` in `v->_TranspRgnBounds[r - 1] >= SoilWatAll.Site._TranspRgnBounds[r]`?

Please add code comment to explain why otherwise simplify to just use `v`